template/tmpl/container.tmpl

```diff
@@ -3,8 +3,8 @@
     docs.forEach(function(doc, i) {
 ?>
 <?js if (doc.kind === 'mainpage' || (doc.kind === 'package')) { ?>
-	<?js if (i === 0 && title !== 'Index'){ ?>
-	<span class="page-title"><?js= title ?></span>
+	<?js if (i === 0){ ?>
+	<h1 class="page-title"><?js= title ?></h1>
```

Comment:
Note:
https://github.com/docstrap/docstrap/pull/90/files

should it be h1 and title !== Index ?